polaris/dataset/_column.py

```diff
@@ -18,6 +18,12 @@ class Modality(enum.Enum):
     IMAGE = "image"
 
 
+class ContentType(str, enum.Enum):
+    """Used to specify column's IANA content type in a dataset."""
+
+    SMILES = Literal["chemical/x-daylight-smiles"]
```

Comment:
**Suggestion**
This type definition might be better defined with others we have in `polaris.utils.types`. We have other types there that leverage `Literal`, so maybe something like:
```python
KnownContentType: TypeAlias = Literal["chemical/x-daylight-smiles"]
ContentType: TypeAlias = KnownContentType | str
```

Also, not sure the enum adds a lot here... Maybe just using the string "chemical/x-daylight-smiles" directly? What do you think?

Reply:
On the use of an enum, one question came to mind: Should a known `content-type` be associated with a modality? If so, having an enum could define this mapping that we could then use in a Pydantic validator for the `ColumnAnnotation` ?